bk-register/ROOTFS/opt/sage/beekeeper/register/register.py

```diff
@@ -77,6 +88,20 @@ def register():
         logger.error(e)
         return "Error: unable to register id [{}]\n".format(id), 500
 
+    # update beekeeper db
+    payload = {"node_id": id, "source": "beekeeper-register", "operation":"insert", "field_name": "registration_event", "field_value": datetime.datetime.now().replace(microsecond=0).isoformat()}
+
+    try:
+        bk_api_response = requests.post(f'{BEEKEEPER_DB_API}/log',data=json.dumps(payload))
+    except Exception as e:
+        return f"Error: X Beekeeper DB API ({BEEKEEPER_DB_API}) cannot be reached: {str(e)}"
```

Comment:
same question (as line 55) about the return error code

Reply:
done